src/devices/Mpr121/Channels.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Mpr121
+{
+    // MPR121 channels.
```

Comment:
I think the comments here and above the values can be removed - they are not super useful anyway 😄

Reply:
Fixed.